Library/Contributions/brew_bash_completion.sh

```diff
@@ -48,6 +48,7 @@ __brewcomp ()
 
     for s in $1; do
         __brewcomp_words_include "$s" && continue
+        [[ $s == instal ]] && continue
```

Comment:
Could you please add a comment above this line explaining what’s this line does? It’s hard to understand it if you don’t know the context.